cloudify_cli/commands/profiles.py

```diff
@@ -385,6 +385,52 @@ def set_cmd(profile_name,
 
 
 @profiles.command(
+    name='set-cluster',
+    short_help='Set connection options for a cluster node')
+@cfy.options.cluster_node_name
```

Comment:
Why is this an option and not an argument?

Reply:
good point, thanks